pkg/qe-tests/cypress/integration/models/plan.ts

```diff
@@ -440,4 +443,22 @@ export class Plan {
       this.finalReviewStep(duplicatePlanData);
     }
   }
+
+  // OCP Migration Network Selection in Plan wizard on General Page
+  selectADifferentNetwork(planData: PlanData): void {
+    const { name, sProvider, tProvider, namespace } = planData;
+    // const { migrationNetwork } = providerData;
+    Plan.openList();
+    clickByText(button, createPlan);
+    this.fillName(name);
+    this.selectSourceProvider(sProvider);
+    this.selectTargetProvider(tProvider);
+    this.selectNamespace(namespace);
+    cy.wait(2 * SEC);
+    clickByText(button, differentNetwork);
+    cy.get(defaultNetwork).should('contain.text', 'ovn-kubernetes1'); //migrationNetworkPlan(also tried)
+    selectFromDroplist(defaultNetwork, podNetwork);
+    confirm();
+    next();
```

Comment:
This method is not required, just add network selection in existing method if another network is defined in the config